XamlStyler.UnitTests/TestFiles/TestProcessingInstructionHandling.expected

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="UTF-8" ?>
+<?xaml-comp compile="true" ?>
+<?Mapping compile="true" ?>
+<ResourceDictionary xmlns="http://xamarin.com/schemas/2014/forms">
+  <ResourceDictionary.MergedDictionaries>
+    <ResourceDictionary Source="Colors.xaml" />
```

Comment:
Can you reduce the test file to include the minimal amount of code to test processor instructions? Maybe it all is required? Not a big deal, but it helps reduce noise when running tests.